.github/ISSUE_TEMPLATE.md

```diff
@@ -1,36 +1,46 @@
-> Please fill out this template when filing an issue.
-> All lines marked as quotes (`>`) instruct you with
-> what info we expect.
-> Remove this opening paragraph.
+> We LOVE to help with any issues or bug you have!
+
+> **Questions**: If you have questions about how to use Realm, please ask on [SO](http://stackoverflow.com/questions/ask?tags=realm) - we monitor the Realm tag.
+
+> **Feature Request**: Just fill in the first two sections below.
+
+> **Bugs**: To help you as fast as possible with an issue or bug please describe your issue and the steps you have taken to reproduce it in as many details as possible.
+>
+> Thanks for helping us help you :-)
+>
+> Please remove this and above before submitting.
 
 ## Goals
 
-> Replace this with what you tried to achieve.
+?
```

Comment:
Some of them might be helpful. I just saw that half of them did actually not add any value at all. E.g. the details under "Steps to reproduce". So my intent was to get rid of the "noise" of text that didn't add useful value. Then I can see some might and we can keep those or we can add it if we see it's a problem? This way I think we can have just the intro and instructions mostly at the top, and after people have read that once, they will just quickly skip it in the future, but the template itself will be easier/lean.
But if there are specific headings were details are needed to explain it we can do that. In Java I made a slightly different version that had the details in the headings instead. Not sure it looks so good now though as the headings are big. But that could be an alternative as well.